docs/api/ethereum.md

```diff
@@ -1,5 +1,645 @@
 # API / Ethereum
 
+## Bitski back-end provider
+
+A [Bitski](https://www.bitski.com/) back-end provider is applied back-end Node.js usage. You will be able to connect to ethereum blockchain and create queries and mutation via your Bitski developer wallets. To setup Bitski back-end provider it is required to first create a [development account](https://developer.bitski.com/) on Bitski.
+
+### BitskiProvider(options)
+
+A `class` providing the communication with the Ethereum blockchain through [Bitski](https://www.bitski.com/).
+
+**Arguments**
+
+| Argument | Description
+|-|-|-
+| options.assetLedgerSource | A `string` representing the URL to the compiled ERC-721 related smart contract definition file. This file is used when deploying new asset ledgers to the network.
+| options.clientId | A `string` representing the Bitski client id. You get the client ID by creating a [developer account](https://developer.bitski.com/) on Bitski.
+| options.credentialsId | A `string` representing the Bitski credentials ID. You get the cretentials ID by creating a [developer account](https://developer.bitski.com/) on Bitski.
+| options.credentialsSecret | A `string` representing the Bitski secret. You get the credentials secret by creating a [developer account](https://developer.bitski.com/) on Bitski.
+| options.mutationTimeout | A `number` representing the number of milliseconds in which a mutation times out. Defaults to `3600000`. You can set it to `-1` for disable timeout.
+| options.orderGatewayId | A `string` representing an Ethereum address of the [order gateway](/#public-addresses).
+| options.networkName | A `string` representing which ethereum network we will connect to.
```

Comment:
`| options.networkName | A `string` representing the Ethereum network we will connect to.`